tests/data/nccm/data.py

```diff
@@ -11,10 +11,15 @@
 from rasterio.crs import CRS
 from rasterio.transform import Affine
 
-SIZE = 32
+SIZE = 1000
 
 np.random.seed(0)
-files = ["CDL2017_clip.tif", "CDL2018_clip1.tif", "CDL2019_clip.tif"]
+files = [
+    "CDL2017_clip.tif",
+    "CDL2018_clip1.tif",
+    "CDL2019_clip.tif",
+    "CDL2022_clip.tif",
```

Comment:
Instead of adding a year of NCCM that doesn't exist, wouldn't it be better to add a year of Sentinel-2 that does exist?